src/Root.tsx

```diff
@@ -71,45 +74,59 @@ type LoginState = 'logging-in' | 'logged-in' | 'unauthorized' | 'error';
 function ContentWrapper({ children }: { children: React.ReactChild }) {
     const authCompleteRouteMatch = useRouteMatch('/auth/complete');
     const [loginState, setLoginState] = useState<LoginState>('logging-in');
-    const navn = Cookies.getNameFromAccessToken();
```

Comment:
Går det att ungå `me.me`? 🙈

Reply:
Ja, vi kan se videre på litt forbedringer i morgen. Merger denne nå så løsningen ikke forblir brukket for lenge.